mix.exs

```diff
@@ -35,7 +35,9 @@ defmodule ExDoc.Mixfile do
 
   defp aliases do
     [
-      clean: [&clean_test_fixtures/1, "clean"]
+      clean: [&clean_test_fixtures/1, "clean"],
+      setup: ["deps.get", &setup_assets/1],
+      docs: ["compile --force", &build_assets/1, "docs"]
```

Comment:
I think we should be able to get rid of `compile --force` if we add `@external_resource ...` to proper modules, I can look into that later.

Reply:
:+1: